src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/Cache/PublicationCacheServiceTests.cs

```diff
@@ -305,6 +305,9 @@ public async Task GetPublicationTree_DataTables_SupersededPublicationWithDataOnL
                     {
                         Title = "Publication A",
                         IsSuperseded = true,
+                        SupersededById = Guid.NewGuid(),
+                        SupersededByTitle = "Publication B",
+                        SupersededBySlug = "publication-b",
```

Comment:
I don't think these need to be added. We filter releases that are superseded in `PublicationCacheService#FilterPublicationTreePublication`, which only uses `IsSuperseded`. By adding these extra fields to the test data, you're implying that these are also used by the method under test. 

I think our general pattern with test data is that we only set values that need to be set, such that someone reading the tests can infer what the method under test actually uses. It's possible other devs disagree, however.